controllers/mirrorpeer_controller.go

```diff
@@ -345,3 +351,53 @@ func (r *MirrorPeerReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		For(&multiclusterv1alpha1.MirrorPeer{}, builder.WithPredicates(mpPredicate)).
 		Complete(r)
 }
+
+func (r *MirrorPeerReconciler) createDRClusters(ctx context.Context, mp *multiclusterv1alpha1.MirrorPeer) error {
+	logger := log.FromContext(ctx)
+	for _, pr := range mp.Spec.Items {
+		clusterName := pr.ClusterName
+		s3SecretName := utils.GetSecretNameByPeerRef(pr, utils.S3ProfilePrefix)
+		rookSecretName := utils.GetSecretNameByPeerRef(pr)
+
+		rs, err := utils.FetchSecretWithName(ctx, r.Client, types.NamespacedName{Name: rookSecretName, Namespace: clusterName})
+		if err != nil {
+			logger.Error(err, "failed to fetch rook secret", "Secret", rookSecretName)
+			return err
+		}
+
+		ss, err := utils.FetchSecretWithName(ctx, r.Client, types.NamespacedName{Name: s3SecretName, Namespace: clusterName})
+		if err != nil {
+			logger.Error(err, "failed to fetch s3 secret", "Secret", s3SecretName)
+			return err
+		}
+
+		rt, err := utils.UnmarshalRookSecret(rs)
+		if err != nil {
+			logger.Error(err, "failed to unmarshal rook secret", "Secret", rookSecretName)
+			return err
+		}
+
+		st, err := utils.UnmarshalS3Secret(ss)
+		if err != nil {
+			logger.Error(err, "failed to unmarshal s3 secret", "Secret", s3SecretName)
+			return err
+		}
+
+		dc := ramenv1alpha1.DRCluster{
+			ObjectMeta: metav1.ObjectMeta{Name: clusterName},
+		}
+
+		_, err = controllerutil.CreateOrUpdate(ctx, r.Client, &dc, func() error {
+			dc.Spec.S3ProfileName = st.S3ProfileName
+			dc.Spec.Region = ramenv1alpha1.Region(rt.FSID)
```

Comment:
Ack! Each DRCluster will carry its region as the Ceph FSID, and hence will be equal for Metro, and different in case of Regional. This will satisfy ramen logic in the interim, where async/sync decisions are made based on the region values.